client/components/data/query-site-stats/index.jsx

```diff
@@ -38,7 +38,7 @@ class QuerySiteStats extends Component {
 
 		this.props.requestSiteStats( siteId, statType, query );
 		this.clearInterval();
-		if ( heartbeat, isAutoRefreshAllowedForQuery( query ) ) {
+		if ( heartbeat && isAutoRefreshAllowedForQuery( query ) ) {
```

Comment:
Yes, I agree, I remember this PR. We explored different directions before settling on the right one. I should have split it into smaller ones.